src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -567,6 +574,33 @@ class ErgoWalletActor(settings: ErgoSettings,
       .toIterator
   }
 
+  /**
+    * Collects boxes according to given request
+    *
+    * @param targetBalance - Balance requested by user
+    * @param targetAssets  - ID's and amounts of other tokens
+    * @return collected ErgoBoxes and ChangeBoxes
+    */
+  private def getRequestBoxes(targetBalance: Long, targetAssets: Map[ErgoBox.TokenId, Long]): Try[CollectedBoxes] = {
```

Comment:
why it is not used in transaction generation methods ? Likely we have boilerplate then